### DISCUSSION TITLE
organization send limit question

### DISCUSSION THREAD

**Discussion by z3d3m0n** [postalserver/postal#2221] [Development discussions]
Hello,

hope everybody is safe and well.

is there a way how to limit the amount of Mail Server a user or organization can create ?
is there a way how to set a limit on Send limit of Mail Servers a user or organization can create ?

eg I want to add a user that will be able to create only two mail servers and each mail server could send 5000 emails per hour max

thxl.

**Comment by willpower232** [comment]
This isn't possible but if you do the creation work first, then you should be able to achieve the same end.

I believe if you create an organisation and then the two mail servers with the limit you desire, then you can create a user for that organisation to access Postal but not be allowed make any more changes at that level.

**Reply by z3d3m0n** [reply]
Thanks for reaching out,
if I create mail servers, set the limitation and provide user access, the user then can create another ten mail servers without the limitation.

How to create user and not allow him to create new mail servers ?

**Reply by willpower232** [reply]
Oh I thought that feature wouldn't be available to regular users as opposed to admin users but I guess its just available all the time. I don't think there is a way to stop it without a code change then.

**Reply by qhgh** [reply]
> Oh I thought that feature wouldn't be available to regular users as opposed to admin users but I guess its just available all the time. I don't think there is a way to stop it without a code change then.

Hi there, I was just wondering if this was possible yet? Currently, a user can log in to the Org and simply create new servers, bypassing any/all restrictions the Admin has set.

Thanks.
